resources/solr-configuration/src/main/resources/patients/conf/schema.xml

```diff
@@ -61,6 +61,7 @@
     <field name="reporter" type="string" indexed="false" stored="true" required="true" />
     <field name="visibility" type="string" indexed="true" stored="true" required="false" />
     <field name="accessLevel" type="int" indexed="true" stored="true" required="false" />
+    <field name="phenotype_ancestors" type="string" indexed="true" stored="true" multiValued="true"/>
```

Comment:
Turns out this is basically redundant with the `extended_phenotype` and `extended_prenatal_phenotype` fields.